### ISSUE TITLE
Fixed header and left column jumps

### ISSUE THREAD

**Issue by kzimic** [lai32290/TableHeadFixer#11]
When the header and left column set to fixed it jumps while using IE and if the Laptop is using battery.
I'm thinking this has todo with relative positioning.

**Comment by kaxiaoche** [comment]
Hi @kzimic ,

Did you fix this bug now ? How did you do ?

: - )

**Comment by kzimic** [comment]
No I didn't fix it. Using fixed and absolute position together will fix the issue when freezing the header but it's not working for the left column...

**Comment by lai32290** [comment]
This problem is occur because plugin re-calculate and re-apply all fixed cells position in scroll event.

When computer has high performance, this operation is so fast, so is look lick normal, but when computer or browser is slow, occur this situation.

In this moment, I really don't know how to can I resolve this problem, but I will continue study about this.

**Comment by Manjunath1847** [comment]
Can you please fix the about issue
it would be helpful at most

**Comment by lai32290** [comment]
I don't really know how to fix this problem now.
If someone has any idea, maybe we can try to fix this
Sorry

**Comment by Manjunath1847** [comment]
Can you please suggest me any other jquery plug-in to freeze table header
and left column.

It will help a lot for me

**Comment by ImaRamos** [comment]
Hello friends,

Someone solved this problem? I really need to make it work in I.E :(

Best Regards

**Comment by Manjunath1847** [comment]
mee too... Please fix it :(

**Comment by ImaRamos** [comment]
The only thing I could do is use a  delay on the scroll function to be less jumpy... nothing else :/

**Comment by lai32290** [comment]
Sorry guys, I really don't know how to fix this problem now, but I'm studying for this.

**Comment by Manjunath1847** [comment]
Thank you... Awaiting for resolution :)

**Comment by Manjunath1847** [comment]
Hi,

  Can i know how you have a used the delay on scroll function.

**Comment by ImaRamos** [comment]
Yes friend, something like that.Debounce it's a jquery function.

     var timer = 100;
    var dprocess = $.debounce(timer, OnScroll);

			if (dprocess !== undefined)
			{
			    parent.scroll(dprocess);
			}

http://benalman.com/code/projects/jquery-throttle-debounce/examples/debounce/

**Comment by Manjunath1847** [comment]
Any update on this please?

**Comment by Manjunath1847** [comment]
Hello, Any update on this issue

**Comment by SainathBommisetty** [comment]
Any update on this please, its very urgent!!

**Comment by itsems** [comment]
+1
sitll need it...
